index.js

```diff
@@ -2,23 +2,29 @@
 
 const Promise = require('bluebird');
 const readInput = Promise.promisify(require('read'));
-const argv = require('yargs').argv;
+const argv = require('yargs').array('domain').argv;
 const xtend = require('xtend');
 const getCertificate = require('./lib.js');
 
-const read = (options) => () => {
+const read = (options) => (forceReadInput = false) => {
```

Comment:
I'm not sure I understand what `forceReadInput` is for, would you mind explaining why it's needed? Unless there are any bugs I'm not aware of, the old implementation worked fine (prompt for values when they are missing or invalid).